config/config.yaml

```diff
@@ -51,57 +51,50 @@ algorithms:
       run1:
         k: range(100,201,100)
 
-      - name: "omicsintegrator1"
-        params:
-              include: true
-              run1:
-                  b: [5, 6]
-                  w: np.linspace(0,5,2)
-                  d: 10
-                  dummy_mode: ["file"] # Or "terminals", "all", "others"
-
-      - name: "omicsintegrator2"
-        params:
-              include: true
-              run1:
-                  b: 4
-                  g: 0
-              run2:
-                  b: 2
-                  g: 3
-
-      - name: "meo"
-        params:
-              include: true
-              run1:
-                  max_path_length: 3
-                  local_search: "Yes"
-                  rand_restarts: 10
-
-      - name: "mincostflow"
-        params:
-              include: true
-              run1:
-                  flow: 1 # The flow must be an int
-                  capacity: 1
-
-      - name: "allpairs"
-        params:
-              include: true
-
-      - name: "domino"
-        params:
-              include: true
-              run1:
-                  slice_threshold: 0.3
-                  module_threshold: 0.05
+  - name: "omicsintegrator1"
+    params:
+      include: true
+      run1:
+        b: [5, 6]
+        w: np.linspace(0,5,2)
+        d: 10
+        dummy_mode: "file" # Or "terminals", "all", "others"
+
+  - name: "omicsintegrator2"
+    params:
+      include: true
+      run1:
+        b: 4
+        g: 0
+      run2:
+        b: 2
+        g: 3
+
+  - name: "meo"
+    params:
+      include: true
+      run1:
+        max_path_length: 3
+        local_search: "Yes"
+        rand_restarts: 10
+
+  - name: "mincostflow"
+    params:
+      include: true
+      run1:
+        flow: 1 # The flow must be an int
+        capacity: 1
+
+  - name: "allpairs"
+    params:
+      include: true
 
   - name: "domino"
```

Comment:
The DOMINO algorithm was dropped? Was that an accident?

Reply:
[I'm seeing it on my fork](https://github.com/Reed-CompBio/spras/blob/7ccae7a567255942ceb029a5daf30efb19047f1e/config/config.yaml#L92-L97) - are we desynced in some way? I seem to be up to date.